word-guess

```diff
@@ -0,0 +1,171 @@
+    puts ' / _   _   _ \ '.colorize(:blue)
+    puts ' |/ \\ / \\ / \\|'.colorize(:blue)
+  end
+
+  def bottom
+    puts  '   o  (_} o '.colorize(:white)
+    puts  '    \\/.X.\\/'.colorize(:white)
+    puts  '      |_|'.colorize(:white)
+    puts  '     // \\\\ '.colorize(:white)
+    puts  '     \\\\ //'.colorize(:white)
+    puts  '      U U'.colorize(:white)
+  end
+  def print_strings
+    @parachute_strings.each do |string|
+      print string.colorize(:magenta)
+    end
+  end
+
+  # def removing_strings
+  #   case
+  #   when @life == 4
+  #     print_strings
+  #   when @life == 3
+  #     @parachute_strings[-1] = " "
+  #     print_strings
+  #   when @life == 2
+  #     @parachute_strings[-1] = " "
+  #     @parachute_strings[-2] = " "
+  #     print_strings
+  #   when @life == 1
+  #     @parachute_strings[-1] = " "
+  #     @parachute_strings[-2] = " "
+  #     @parachute_strings[-4] = " "
+  #     print_strings
+  #   when @life == 0
+  #     @parachute_strings[-1] = " "
+  #     @parachute_strings[-2] = " "
+  #     @parachute_strings[-4] = " "
+  #     @parachute_strings[0] = " "
+  #     print_strings
+  def remove1
+    @parachute_strings[-1] = " "
+    print_strings
+  end
+  def remove2
+    @parachute_strings[-1] = " "
+    @parachute_strings[-2] = " "
+    print_strings
+  end
+  def remove3
+    @parachute_strings[-1] = " "
+    @parachute_strings[-2] = " "
+    @parachute_strings[-4] = "   "
+    print_strings
+  end
+  def remove4
+    @parachute_strings[-1] = " "
+    @parachute_strings[-2] = " "
+    @parachute_strings[-4] = "   "
+    @parachute_strings[0] = "    "
+    print_strings
+  end
+
+  # end
+
+end
+
+
+class Game
+  attr_accessor :secret_word_array, :user_input, :life, :correct_guesses_array, :picture, :incorrect_guesses
+
+  def initialize(secret_word, picture, life)
+    @picture = picture
+    @secret_word_array = secret_word
+    @user_input = 0
+    @incorrect_guesses = []
+    @correct_guesses_array = Array.new(@secret_word_array.length, "_")
+    @life = life
+  end
+
+  # Method to compare user_input to secret_word
+  def compare(guess)
+    @user_input = guess
+    @secret_word_array.each do |letter|
+      if letter == @user_input
+        @correct_guesses_array[@secret_word_array.index(letter)] = letter
+      end
+    end
+    if @correct_guesses_array.include?(@user_input) == false
+      @incorrect_guesses << @user_input
+
+      puts "\n False guesses :#{@incorrect_guesses}"
+      @picture.top
+      case @life
+      when 4
+        @picture.remove1
+      when 3
+        @picture.remove2
+      when 2
+        @picture.remove3
+      when 1
+        @picture.remove4
+        # @picture.removing_strings
+      end
+      @life -= 1
+      puts
+      @picture.bottom
+    end
+  end
+end
+word_list = ["cat", "dog", "horse", "bird"]
+
+
+parachute_man = Picture.new()
+new_game = Game.new(word_list.sample.split(""), parachute_man, parachute_man.life )
+puts
```

Comment:
Since the initial life count comes from the picture, it's a little redundant to pass both into the constructor here.